README.md

```diff
@@ -1 +1,147 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> For this project I used basic HTML and CSS.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Basic HTML & CSS portfolio using GitHub.
+
+- **Linter Usage**
+- **GitHub Workflow**
+- **HTML, CSS and JS**
+- **Flexbox**
+- **Grid**
+- **Responsive design**
+- **Accessibility support**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://masterleiva.github.io/Portfolio/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+- Computer
+- Web Browswer
+- Internet Connection
+
+### Setup
+
+Make sure you have Git and GitHub [setup](https://www.theodinproject.com/lessons/foundations-setting-up-git).
+
+### Install
+
+Install this project with:
+```sh
+  cd my-folder
+  git clone ssh://user@server:/GitRepos/myproject.git
+```
+
+### Usage
+
+To run the project, open the html file on your repository using your web browser.
+
+### Run tests
+
+To run tests, go to the github page of your repository and then go to actions, to make use of the linter.
+
+### Deployment
+
+N/A
+
+## 👥 Authors <a name="authors"></a>
+
+> This is a personal project.
+
+👤 José De La Rosa
+
+- GitHub: [@githubhandle](https://github.com/jrraul4)
+- Twitter: [@twitterhandle](https://twitter.com/joserauldelaro1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/jose-raul-de-la-rosa-98aa70265/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> More coming soon:
+- [ ] **Pop up window**
+- [ ] **Contact form validations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just send a message and maybe add new things.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to Microverse and to my coding partners for help me in this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The name of the LICENSE file in your directory is LICENSE, so use `[MIT](./LICENSE)` instead of `[MIT](./MIT.md)` to avoid the 404 error.